modules/core/src/trace.cpp

```diff
@@ -1012,6 +1012,8 @@ void parallelForFinalize(const Region& rootRegion)
             }
         }
     }
+    (void) threads; // To avoid -Wunused-but-set-variable
```

Comment:
Thank you for your comment, I agree with you ! I will remove `threads` variable.  And I'm sorry to commit fixed code needs some time because I will test  compiling with several version of gcc and clang.